.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+## User Story
+As X, I want Y so that Z.
+
+## Impacts
+Ex. API, UI, Docs
+
+## Role
+
+
+## Assumptions
+-
+-
+
+## API Details

```

Comment:
Not important, but for this repo we wont likely have APIs defined like this or UI Details needed. Maybe just *Implementation Details*.